src/docfx/lib/log/Telemetry.cs

```diff
@@ -283,6 +301,12 @@ public static void Flush()
         Task.WaitAny(Task.Run(s_telemetryClient.Flush), Task.Delay(10000));
     }
 
+    public static void Dispose()
+    {
```

Comment:
I think we can skip this implementation as all resources are static.

Reply:
Removed